example/config.js

```diff
@@ -7,12 +7,20 @@ var System = (function() {
   // Create instance of bitimports
   var importer = bitimports.config({
     "paths": {
-      "babel": "../node_modules/babel-bits/dist/index.min.js",
-      "sass": "../node_modules/sassy-bits/dist/index.min.js",
-      "plugin1": "./js/plugin1.js"
-    }
+      "babel": "/node_modules/babel-bits/dist/index.min.js",
+      "sass": "/node_modules/sassy-bits/dist/index.min.js",
```

Comment:
Great! It's very slow so we don't want to use this unless we absolutely must :)  this is also a reference implementation for a SCSS plugin you had asked me to do.